examples/cifar_cnntransformer/README.md

```diff
@@ -3,17 +3,20 @@
 ### 1. Description
 In this example we train a standard 8 layer CNN on CIFAR10 as a baseline. We then take the same CNN architecture and stack a Transformer-Encoder ontop and train this new CNNTransformer model from scratch. We present several different variants of this model where we only alter the Transformer size. Below, the validation accuracy of each model is compared.
 
-![Model Comparisons](CIFAR10-ModelComparison-ValAcc.png)
-(The code example on this page does not aim to show any meaningful results, but only demonstrate the use of the CNNTransformer)
+[//]: # (![Model Comparisons]&#40;CIFAR10-ModelComparison-ValAcc-Lightning.png&#41;)
+<div align="center">
+	<img src="CIFAR10-ModelComparison-ValAcc-Lightning.png" alt="Editor" width="80%">
+</div>
+(The code example in this folder is not intended to achieve any state-of-art result, but rather to demonstrate the use of CNNTransformer.)
```

Comment:
@HaolinWang2001 
- This is quite verbose. Just say "This example is not inteneded" rather than "The code example in this folder is not indended".
- I said "state-of-the-art" but you put "state-of-art", have you checked your changes against my suggestion? Why are you removing "the"? Have you googled to learn or asked Xianyuan if you are not sure?

Reply:
Changes have been made. I have double-checked that "state-of-art" is a typo, which has been corrected. Thank you for bringing this to my attention, I will take extra care to ensure that such errors do not occur again in the future.